ldap/servers/slapd/back-ldbm/ldbm_entryrdn.c

```diff
@@ -992,6 +992,7 @@ entryrdn_lookup_dn(backend *be,
     ID workid = id; /* starting from the given id */
     rdn_elem *elem = NULL;
     int maybesuffix = 0;
+    ID suffix_id;
```

Comment:
Style: We usually initialize the variable at declaration time. 
 ID suffix_id = 1;